[32mINFO    [0m root:basepage.py:32 打开网页https://www.imymac.com/store/
[32mINFO    [0m root:basepage.py:226 关闭黑色星期五优惠链接
[32mINFO    [0m root:basepage.py:153 使用js操控浏览器==》document.querySelector('div.close-tag').click()
[32mINFO    [0m root:basepage.py:90 点击元素：('css', 'body>div.storeBG>div>div>div:nth-child(1)>div:nth-child(2)>a:nth-child(6)')
[32mINFO    [0m root:basepage.py:90 点击元素：('css', 'body>div.buyBg>div>div>div:nth-child(3)>p:nth-child(2)>span:nth-child(3)')
[32mINFO    [0m root:basepage.py:90 点击元素：('css', 'body>div.buyBg>div>div>div:nth-child(3)>p:nth-child(2)>span:nth-child(1)')
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023B8ED66820>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/36a2a6933c874cebaeacf7786b334b5b/timeouts
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023B8EE62FD0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/36a2a6933c874cebaeacf7786b334b5b/timeouts
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023B8EE62E80>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/36a2a6933c874cebaeacf7786b334b5b/timeouts
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023B8EDC2BE0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/36a2a6933c874cebaeacf7786b334b5b
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023B8F13ADF0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/36a2a6933c874cebaeacf7786b334b5b
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:780 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023B8F13ABE0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。')': /session/36a2a6933c874cebaeacf7786b334b5b